razorpay/razorpay.php

```diff
@@ -24,7 +24,7 @@ public function __construct()
         $this->name = 'razorpay';
         $this->displayName = 'Razorpay';
         $this->tab = 'payments_gateways';
-        $this->version = '2.0.0';
+        $this->version = '2.0.1';
```

Comment:
Are we doing a version bump in this PR? Or is this a mistake?

Reply:
In this only, as we are already created beta release with that version bump.

Reply:
Shouldn't this be 2.0.1-beta then?

Reply:
not sure, because last beta release, we did just as pre-release. If we are going to make beta as public release, then we can.